pkg/services/hcloud/server/server.go

```diff
@@ -97,6 +99,9 @@ func (s *Service) Reconcile(ctx context.Context) (res reconcile.Result, err erro
 	if server == nil {
 		server, err = s.createServer(ctx)
 		if err != nil {
+			if errors.Is(err, errServerCreateNotPossible) {
+				return reconcile.Result{RequeueAfter: 5 * time.Minute}, nil
```

Comment:
I think 5 minutes are a bit long. What about three minutes?

Reply:
I think 5 min are a good number, considering the fact that manual intervention of users is required. they might not see this for hours. 

If the user wants a more responsive system, they can reduce the sync period as well.

@batistein ?

Reply:
5 min are enough to be responsive enough. From my understanding this case only happens when something is misconfigured right?